isthmus/src/test/java/io/substrait/isthmus/StringFunctionTest.java

```diff
@@ -0,0 +1,66 @@
+package io.substrait.isthmus;
+
+import java.util.List;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class StringFunctionTest extends PlanTestBase {
+
+  static List<String> CREATES = List.of("CREATE TABLE strings (c16 CHAR(16), vc32 VARCHAR(32))");
+
+  @ParameterizedTest
+  @ValueSource(strings = {"c16", "vc32"})
+  void char_length(String column) throws Exception {
+    String query = String.format("SELECT char_length(%s) FROM strings", column);
+    assertSqlSubstraitRelRoundTrip(query, CREATES);
+  }
+
+  @ParameterizedTest
+  @ValueSource(strings = {"vc32"})
```

Comment:
Using the `||` concatenation operator fails on CHAR columns with "`IllegalArgumentException: Unable to convert call ||(char<16>?, char<16>?) at RexExpressionConverter.visitCall(RexExpressionConverter.java:83)`".
This should be investigated.